mne_features/univariate.py

```diff
@@ -28,7 +28,43 @@ def get_univariate_funcs(sfreq):
     -------
     univariate_funcs : dict
     """
+<<<<<<< HEAD
```

Comment:
A conflict remains!
You should keep ``return _get_feature_funcs(sfreq, __name__)`` and remove the rest.